wp-content/themes/pub/wporg-learn-2024/src/sensei-meta-list/index.php

```diff
@@ -90,6 +90,48 @@ function render( $attributes, $content, $block ) {
 				'key'   => 'last-updated',
 			),
 		);
+	} else if ( 'lesson-plan' === $block->context['postType'] ) {
+		$lesson_plan_id = $block->context['postId'];
+
+		$duration = get_post_taxonomy_terms_links( $lesson_plan_id, 'duration' );
+		$audience = get_post_taxonomy_terms_links( $lesson_plan_id, 'audience' );
+		$level = get_post_taxonomy_terms_links( $lesson_plan_id, 'level' );
```

Comment:
These links are in this format https://learn.wordpress.org/level/beginner/, I think they should be https://learn.wordpress.org/lesson-plans/?wporg_lesson_level=beginner

Reply:
Same goes for duration, audience and type. My expectation would be that I'm filtering Lesson Plans by that field, rather than all content types.

Thoughts @fcoveram ?

Reply:
+1 to your suggestion. I would expect the same.

> so that the user lands on the filtered Lesson Plan archive.

Just to double check. You mean [this page](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=3873-12813&m=dev) but on its "Lesson version" (aka, without the colored hero and pagination at the end), right? If so, then we're aligned.

Reply:
Correct, looks like [this](https://learn.wordpress.org/lesson-plans/?wporg_lesson_level=beginner&new-theme=1)

![Screenshot 2024-06-25 at 8 17 37 AM](https://github.com/WordPress/Learn/assets/1017872/7b6687cb-1e5a-4234-a8e2-93769d206bef)

Reply:
That's perfect

Reply:
I currently only allow 'level' to maintain links. See https://github.com/WordPress/Learn/pull/2540#discussion_r1669531408

![image](https://github.com/WordPress/Learn/assets/18050944/5feaf511-5e4a-453c-be31-02b058e67067)
![image](https://github.com/WordPress/Learn/assets/18050944/7f1abf3e-1861-4c4b-84e8-6c1fafa55e33)